playbackScreen/src/main/kotlin/voice/playbackScreen/BookPlayController.kt

```diff
@@ -231,6 +233,16 @@ class BookPlayController(bundle: Bundle) : ViewBindingController<BookPlayBinding
       .showDialog(router)
   }
 
+  private fun openPlaybackSpeedDialog(speed: Float) {
+    activity?.showAsBottomSheet {
```

Comment:
What is the reason that this is not just a regular composable like the other dialogs?